src/main/java/mate/academy/springlibrary/controller/AuthController.java

```diff
@@ -19,11 +22,18 @@
 public class AuthController {
 
     private final UserService userService;
+    private final AuthenticationService authenticationService;
 
     @PostMapping("/registration")
     @Operation(summary = "Register user")
     public UserResponseDto register(@RequestBody @Valid UserRegistrationRequestDto userDto) {
         return userService.register(userDto);
     }
 
+    @PostMapping("/login")
+    @Operation(summary = "Login user")
+    public UserTokenResponseDto login(@RequestBody @Valid UserLoginRequestDto userDto) {
+        return authenticationService.authenticate(userDto);
+    }
+
 }
```

Comment:
In the future, please try to keep only changes related to pull request when you push it. It's ok for now but it might make review process more difficult